### ISSUE TITLE
Learning rate warm up not linear increasing

### ISSUE THREAD

**Issue by mauku** [ylabbe/cosypose#42]
Hello, 

I am currently training a single view pose estimation model for an own set of objects following the tless pbr model. However when I compare the learning rates during the warmup phase, which is set in both cases to 50 epochs, there is a difference. The learning rate in my model is not increasing linear during the warmup phase, whereas in your trained tless pbr coarse model the learning rate increases linear during the warmup phase. Is there an additional parameter to set how the learning rate should increase during warmup phase ? After the warmup phase the learning rate is the same for both models.

Thanks in advance!
